.github/workflows/tests-windows.yml

```diff
@@ -10,23 +10,17 @@ on:
 jobs:
   tests:
     runs-on: windows-latest
-    timeout-minutes: 15
+    timeout-minutes: 30
     steps:
       - uses: actions/checkout@v3
       - run: git fetch --depth=1
       - uses: actions/setup-node@v3
         with:
           node-version: 18
+          cache: 'yarn'
       - run: yarn install --frozen-lockfile
-      - run: yarn test:unit --continue -- -- -- --json --outputFile=unit-test-output.json
-      - run: yarn test:integration -- -- -- --json --outputFile=int-test-output.json
```

Comment:
can we keep the `concurrency=1` and revert the output file things as before?

Reply:
I feel we can do without archiving the results as the test output is in the console already. I generally just go through that instead of downloading from the archive. But I don't have a strong opinion on this.

Reply:
We added this initially to run some analysis on flaky windows tests, but I don't think we ever did that.